src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/IntegrationTestBase.kt

```diff
@@ -128,6 +130,8 @@ abstract class IntegrationTestBase {
     recallType = RecallType.FIXED,
     agreeWithRecallRecommendation = agreeWithRecallRecommendation,
     recallLength = recallLength,
+    lastReleasePrison = "prison",
```

Comment:
I think this is an example of why we should use random or at least variable data for testing.
In fact I'm surprised that the IDE does not give a warning - that the `` parameter to the call is un-used.  There may be something I don't know going on with default values in Kotlin here.  
Certainly I think we're being inconsistent in this function now; I'll try to call you.